packages/accordion/src/useAccordion.ts

```diff
@@ -5,49 +5,83 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import { useState } from 'react';
+import { useState, HTMLProps } from 'react';
 import {
   composeEventHandlers,
   generateId,
   KEY_CODES,
   getControlledValue
 } from '@zendeskgarden/container-utilities';
 
+export interface IUseAccordionProps {
+  idPrefix?: string;
+  expandedSections?: number[];
+  onChange?: (expanded: number[]) => any;
+  expandable?: boolean;
+  collapsible?: boolean;
+}
+
+interface IHeaderProps extends HTMLProps<any> {
+  ariaLevel?: number | null;
+  role?: any;
```

Comment:
Without this line, TypeScript complained that `null` cannot a value passed to `role`. 

The `useAccordion` API currently uses `null` to determine whether or not to leave off ARIA attributes. To allow `null` to be passed, I added `role?: any;` to these interfaces.

Reply:
Could this be `role?: string | null;` like the `ariaLevel` type?

Reply:
I tried, but TypeScript doesn't seem to be happy. I believe the reason might be that `role` is already defined in `@types/react/index.d.ts`. Like `role`, this caveat also applies to `tabIndex`.

I think `ariaLevel` was okay, because was not defined. The [type definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1546) has it as `aria-level`, and not `ariaLevel`.

Reply:
Leaving this as is, per another discussion thread in this PR. Thanks for all the input!